### PR TITLE
Improve documentation for some API messages and methods

### PR DESCRIPTION
**Pull request by htdvisser**
#### Summary

This PR adds improvements to some of the Registry API methods and messages.

Refs #252 

#### Changes

- Added/improved comments in proto files
- Regenerate

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [x] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: api/end_device_services.proto [hunk 1]
```diff
@@ -42,20 +45,27 @@ service EndDeviceRegistry {
 
   rpc GetIdentifiersForEUIs(GetEndDeviceIdentifiersForEUIsRequest) returns (EndDeviceIdentifiers);
```
Comment (benolayinka) [inline api/end_device_services.proto:47]: is this one missing?

**Review by benolayinka — APPROVED** [review]
you're a saint 🙌🏽

**Comment by benolayinka** [comment]
A couple apis i didn't see in the docs https://thethingsstack.io/latest/reference/api
Client
Oauth

**Comment by htdvisser** [comment]
I have a separate issue for documenting OAuth clients: #1784

**Pull request merged**